---
layout: page
tags: [Jekyll, HADDOCK, Bonvin, Docking, Simulation, Molecular Dynamics, Structural Biology, Computational Biology, Modelling, Protein Structure]
modified: 2014-08-08T20:53:07.573882-04:00
comments: false
image:
  feature: pages/banner_software.jpg
---

<H1><FONT COLOR=RED>HADDOCK2.2</FONT></H1>

<H2 align=right><FONT COLOR=RED>H</FONT>ADDOCK-related publications</H2>

<hr>
This page lists haddock development articles.
<BR><BR>
The original JACS paper, together with an introduction to the study of biomolecular complexes and an application to the UbcH5-CNOT4 complex can be found online in Cyril Dominguez's Ph.D. thesis:
<ul>
<li><a target=new href="http://www.library.uu.nl/digiarchief/dip/diss/2004-0616-081333/inhoud.htm">
"NMR-based docking of protein-protein complexes:<br> 
the human UbcH5b-CNOT4 ubiquitination complex"</a><br>
Cyril Dominguez, Ph.D. Thesis, June 2004, ISBN 90-393-3721-7.
</ul>
<BR>
<b><i>When using HADDOCK cite:</i></b>
<ul>
<LI>Cyril Dominguez, Rolf Boelens and Alexandre M.J.J. Bonvin (2003). 
<a target=new href="https://doi.org/doi:10.1021/ja026939x">HADDOCK: a protein-protein docking approach based on biochemical and/or biophysical information.</a><br>
    <i>J. Am. Chem. Soc.</i> <b>125</b>, 1731-1737.</LI>
<BR>
<li>S.J. de Vries, A.D.J. van Dijk, M. Krzeminski, M. van Dijk, A. Thureau, V. Hsu, T. Wassenaar and A.M.J.J. Bonvin<br>
"<A target=new HREF="https://doi.org/doi:10.1002/prot.21723">HADDOCK versus HADDOCK: New features and performance of HADDOCK2.0 on the CAPRI targets.</a>"<br>
<EM>Proteins: Struc. Funct. & Bioinformatic</EM> <STRONG>69</STRONG>, 726-733 (2007).
</LI>
</UL>
<BR>
<b><i>And, if used, cite depending on the type of data and setting used the following publications:</i></b>
<BR><BR>
The use of <i>residual dipolar couplings</i> in HADDOCK is described in:<br>
<ul>
<LI>A.D.J. van Dijk, D. Fushman and A.M.J.J. Bonvin (2005)<br>
"<A target=new HREF="https://doi.org/doi:10.1002/prot.20476">Various strategies of using residual dipolar 
couplings in NMR-driven protein docking: Application to Lys48-linked di-ubiquitin and validation against 15N-relaxation data.</A>"<BR>
<EM>Proteins: Struc. Funct. & Bioinformatics</EM>, <STRONG>60</STRONG>, 367-381.
</ul>
<BR>

The use of <i>diffusion anisotropy data</i> in HADDOCK is described in:<br>
<ul>
<li>A.D.J. van Dijk, R. Kaptein, R. Boelens and A.M.J.J. Bonvin<br>
"<A target=new HREF="https://doi.org/doi:10.1007/s10858-006-0024-8">Combining NMR relaxation with chemical shift perturbation data to drive protein-protein docking</A>"<br>
<EM>J. Biomol. NMR</EM>, <STRONG>34</STRONG>, 237-244 (2006).
</ul>
<BR>

The use of <i>pseudo contact shift restraints</i> in HADDOCK is described in:<br>
<ul>
<li>C. Schmitz and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1007/s10858-011-9514-4">Protein-Protein HADDocking using exclusively Pseudocontact Shifts.</a>"<BR>
<EM>J. Biomol. NMR</EM>, <strong>50</strong>, 263-266 (2011).
</ul>
<BR>

The use of <i>SAXS data to score HADDOCK models</i> is described in:<br>
<ul>
<li>E. Karaca and <font color="#333333"><b>A.M.J.J. Bonvin</b></font>.<BR>
"<a target=new href="https://doi.org/doi:10.1107/S0907444913007063">On the usefulness of Ion Mobility Mass Spectrometry and SAXS data in scoring docking decoys.</A>"<BR>
<EM>Acta Cryst. D.</EM>, <STRONG>D69</STRONG>, 683-694 (2013).
</ul>
<BR>

The <i>quantitative use of chemical shifts to score HADDOCK models</i> is described in:<br>
<ul>
<li>D. Stratman, R. Boelens and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1002/prot.23090">Quantitative use of chemical shifts for the modelling of protein complexes.</a>"<BR>
<EM>Proteins: Struc. Funct. & Bioinformatic</EM>, <strong>79</strong>, 2662-2670 (2011).
</ul>
<BR>

<i>Protein-peptide docking</i> with HADDOCK is described in:<br>
<ul>
<li>M. Trellet, A.S.J. Melquiond and <font color="#333333"><b>A.M.J.J. Bonvin</b></font>.<BR>
"<a target=new href="http://dx.plos.org/10.1371/journal.pone.0058769">A Unified Conformational Selection and Induced Fit Approach to Protein-Peptide Docking.</a><br>
<EM>PLoS ONE</EM>, <STRONG>8(3)</STRONG> e58769 (2013).
</ul>
<BR>

<i>Solvated protein-protein docking</i> using HADDOCK is described in:<br>
<ul>
<li>A.D.J. van Dijk and A.M.J.J. Bonvin<br>
"<A target=new HREF="https://doi.org/doi:10.1093/bioinformatics/btl395">Solvated docking: introducing water i
nto the modelling of biomolecular complexes.</a>"<br>
<EM>Bioinformatics</EM>,  <STRONG>22</STRONG> 2340-2347 (2006).
</ul>
<BR>

<i>Solvated protein-protein docking with Kyte-Doolittle propensities </i> using HADDOCK is described in:<br>
<ul>
<li>P.L. Kastritis, K.M. Visscher, A.D.J. van Dijk and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1002/prot.24210">Solvated docking using Kyte-Doolittle-based water propensities.</a>"<BR>
<EM>Proteins: Struc. Funct. & Bioinformatic.</EM>, <STRONG>81</STRONG>, 510-518 (2013).
</ul>
<BR>

<i>Solvated protein-DNA docking </i> using HADDOCK is described in:<br>
<ul>
<li>M. van Dijk, K. Visscher, P.L. Kastritis and <font color="#333333"><b>A.M.J.J. Bonvin</b></font>.<BR>
"<a target=new href="https://doi.org/doi:10.1007/s10858-013-9734-x">Solvated protein-DNA docking using HADDOCK.</a>"<BR>
<EM>J. Biomol. NMR</EM>, <STRONG>56</STRONG>, 51-63 (2013).</ul>
<BR>

<i>Flexible protein-DNA docking</i> using HADDOCK is described in:
<ul>
<li>M. van Dijk, A.D.J. van Dijk, V. Hsu, R. Boelens and  A.M.J.J. Bonvin<br>
"<A target=new HREF="https://doi.org/doi:10.1093/nar/gkl412">Information-driven Protein-DNA Docking using HADDOCK: it is a matter of flexibility.</a>"<br>
<EM>Nucl. Acids Res.</EM>, <STRONG>34</STRONG> 3317-3325 (2006).
<BR><BR>
<li>M. van Dijk and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1093/nar/gkq222">Pushing the limits of what is achievable in protein-DNA docking. Benchmarking HADDOCK's performance.</a>"<BR>
<EM>Nucl. Acid Res.</EM>, <STRONG>38</STRONG>, 5634-5647 (2010).
</ul>
<BR>

<i>(Flexible) multibody docking </i> using HADDOCK is described in:<br>
<ul>
<li>E. Karaca and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1016/j.str.2011.01.014">A  multi-domain flexible docking approach to deal with large conformational changes in the modeling of biomolecular complexes.</a>"<BR>
<EM>Structure</EM>, <STRONG>19</STRONG> 555-565(2011).
<BR>
<li>E. Karaca, A.S.J. Melquiond, S.J. de Vries, P.L. Kastritis and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1074/mcp.M000051-MCP201">Building macromolecular assemblies by information-driven docking: Introducing the HADDOCK multi-body docking server.</a>"<BR>
<EM>Mol. Cell. Proteomics</EM>, <STRONG>9</STRONG>, 1784-1794 (2010).
</ul>
<BR>

<i>Our results in <a target=new href="http://capri.ebi.ac.uk">CAPRI</a></i>, the blind docking experiment, 
are described in (Note that entire issues of Protein are dedicated to CAPRI):<br>
<ul>
<li>J.P.G.L.M. Rodrigues, A.S.J. Melquiond, E. Karaca, M. Trellet, M. Van Dijk, G.C.P. Van Zundert, C. Schmitz, S.J. de Vries, A. Bordogna, L. Bonati, P.L. Kastritis and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1002/prot.24382">Defining the limits of homology modelling in information-driven protein docking</a>"<BR>
<EM>Proteins: Struc. Funct. & Bioinformatics</EM>, <STRONG>81</STRONG>, 2119-2128 (2013).
<BR>
<li>S.J. de Vries, A.S.J. Melquiond, P.L. Kastritis, E. Karaca, A. Bordogna, M. van Dijk, J.P.G.L.M. Rodrigues and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><br>
"<a target=new href="https://doi.org/doi:10.1002/prot.22814">Strengths and weaknesses of data-driven docking in CAPRI</a>"<BR>
<EM>Proteins: Struc. Funct. & Bioinformatic</EM>, <STRONG>78</STRONG>, 3242-3249 (2010).
<BR>
<li>S.J. de Vries, A.D.J. van Dijk, M. Krzeminski, M. van Dijk, A. Thureau, V. Hsu, T. Wassenaar and <font color="#333333"><b>A. M. J. J. Bonvin</b></font><br>
"<A target=new HREF="https://doi.org/doi:10.1002/prot.21723">HADDOCK versus HADDOCK: New features and performance of HADDOCK2.0 on the CAPRI targets.</a>"<br>
<EM>Proteins: Struc. Funct. & Bioinformatic</EM>, advanced online publication (2007).
<BR>
<li>A.D.J. van Dijk, S.J. de Vries, C. Dominguez, H. Chen, H.-X. Zhou and <font color="#333333"><b>A
.M.J.J. Bonvin</b></font><br>
"<A target=new HREF="https://doi.org/doi:10.1002/prot.20563">Data-driven docking: HADDOCKs adventures in CAPRI."
</A><br>
<EM>Proteins: Struc. Funct. & Bioinformatics</EM>, <STRONG>60</STRONG>, 232-238 (2005).
<BR>
</ul>
<BR>

Reviews on data-driven docking:
<ul>
<li>J.P.G.L.M Rodrigues and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1111/febs.12771">Integrative computational modeling of protein interactions.</a>"<BR>
<EM>FEBS J.</EM>, <STRONG>281</STRONG>, 1988-2003 (2014).
<BR>
<li>H. van Ingen and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR> 
"<a target=new href="https://doi.org/10.1016/j.jmr.2013.10.021">Information-driven modeling of large macromolecular assemblies using NMR data.</a>"<BR>
<EM>J. Magn. Reson.</EM> <STRONG>241</STRONG>, 103-114 (2014).
<BR>
<li>E. Karaca and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="https://doi.org/doi:10.1016/j.ymeth.2012.12.004">Advances in integrated modelling of biomolecular complexes.</a>"<BR>
<EM> Methods</EM>, <STRONG>59</STRONG>, 372-381 (2013).
<BR>
<li>P.L. Kastritis and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="http://bookshop.cabi.org/?site=191&page=2633&pid=2443">Predicting and Dissecting High-order Molecular Complexity by Information-driven Biomolecular Docking</a>"<BR>
In: <i>Antimicrobial Drug Discovery. Emerging Strategies</i> Adv. Mol. Cell. Microbiol., Eds G. Tegos and E. Mylokanis, CABI, <STRONG>22</STRONG>, 232-246 (2012).
<BR>
<li>A.S.J. Melquiond, E. Karaca, P.L. Kastritis and <font color="#333333"><b>A.M.J.J. Bonvin</b></font><BR>
"<a target=new href="http://onlinelibrary.wiley.com/doi/10.1002/wcms.91/full">Next challenges in protein-protein docking: From proteome to interactome and beyond.</a>"<BR>
<EM>WIREs Computational Molecular Science</EM>, <strong>2</strong>, 642-651 (2012).
<BR>
<li>Aalt D.J. van Dijk, Rolf Boelens and Alexandre M.J.J. Bonvin (2005).
"<A target=new HREF="https://doi.org/doi:10.1111/j.1742-4658.2004.04473.x">Data-driven docking for the study
of biomolecular complexes.</A>"<BR>
<i>FEBS Journal</i>, <b>272</b>, 293-312.
</ul>
<hr>
